resolwe_bio/processes/workflows/bbduk_star_featurecounts_qc.py

```diff
@@ -0,0 +1,802 @@
+
+        if inputs.alignment.filtering_options.out_score_min:
+            input = {
+                "filtering": {
+                    "out_score_min": inputs.alignment.filtering_options.out_score_min,
+                }
+            }
+            input_star.update(input)
+
+        if inputs.alignment.filtering_options.out_mismatch_nrl_max:
+            input = {
+                "filtering": {
+                    "out_mismatch_nrl_max": inputs.alignment.filtering_options.out_mismatch_nrl_max,
+                }
+            }
+            input_star.update(input)
+
+        if inputs.alignment.alignment_options.align_overhang_min:
+            input = {
+                "alignment": {
+                    "align_overhang_min": inputs.alignment.alignment_options.align_overhang_min
+                }
+            }
+            input_star.update(input)
+
+        if inputs.alignment.alignment_options.align_sjdb_overhang_min:
+            input = {
+                "alignment": {
+                    "align_sjdb_overhang_min": inputs.alignment.alignment_options.align_sjdb_overhang_min
+                }
+            }
+            input_star.update(input)
+
+        if inputs.alignment.alignment_options.align_intron_size_min:
+            input = {
+                "alignment": {
+                    "align_intron_size_min": inputs.alignment.alignment_options.align_intron_size_min
+                }
+            }
+            input_star.update(input)
+
+        if inputs.alignment.alignment_options.align_intron_size_max:
+            input = {
+                "alignment": {
+                    "align_intron_size_max": inputs.alignment.alignment_options.align_intron_size_max
+                }
+            }
+            input_star.update(input)
+
+        if inputs.alignment.alignment_options.align_gap_max:
+            input = {
+                "alignment": {
+                    "align_gap_max": inputs.alignment.alignment_options.align_gap_max,
+                }
+            }
+            input_star.update(input)
+
+        if inputs.alignment.output_options.out_rg_line:
+            input = {
+                "output_options": {
+                    "out_rg_line": inputs.alignment.output_options.out_rg_line
+                }
+            }
+            input_star.update(input)
+
+        alignment = Data.create(
+            process=BioProcess.get_latest(slug="alignment-star"),
+            input=input_star,
+        )
+
+        input_featurecounts = {
+            "aligned_reads": alignment,
+            "n_reads": inputs.quantification.n_reads,
+            "assay_type": inputs.quantification.assay_type,
+            "annotation": inputs.quantification.annotation,
+            "feature_class": inputs.quantification.feature_class,
+            "feature_type": inputs.quantification.feature_type,
+            "id_attribute": inputs.quantification.id_attribute,
+        }
+
+        if inputs.quantification.cdna_index:
```

Comment:
Add an additional check here for the `cdna_index` input. If the `cdna_index` is not given when the `assay_type` is set to auto, fail the workflow. Add this check to the beginning of the workflow, so no individual processes are triggered if this state is detected.